rfcs/README.md

```diff
@@ -12,51 +12,86 @@ or rejected for implementation in oneTBB.
 
 The possible RFC states are:
 
-1. Initial 
-2. Proposed
-3. Experimental
-4. Supported
-5. Archived
+1. Proposed
+2. Experimental
+3. Supported
+4. Archived
 
 Most modifications or new features will naturally start as a part of a 
 GitHub issue or discussion. Small changes do not require a formal RFC. 
 However, if the issue or discussion results in an idea for a significant 
 change or new feature that affects the library's public API or architecture, 
-we recommend opening a PR to add a new RFC to the `rfcs/proposed` directory. 
-The RFC should provide a detailed description and design of the proposed feature.
-or new feature that significantly impacts the library's public API or 
-architecture, it will be suggested that a PR be opened to add a new rfc 
-to the `rfcs/proposed` directory. The RFC contains a more detailed description
-and design for the feature.
+we recommend opening a PR to add a new RFC to the `rfcs/proposed` directory.
+
+The `rfcs/proposed` directory contains RFCs for approved proposals
+that need to be implemented. These documents describe the overall design
+and API for the proposed functionality.
```

Comment:
Do we want to state somewhere that the RFC process (and the `proposed` directory) can be used also for proposals containing only the idea and extended motivation, without stating the design and current API (at least on the initial stage)?

Or the idea is to use discussions for such cases?